docs/running_the_federation.rst

```diff
@@ -9,13 +9,737 @@ Run the Federation
+    - :ref:`establishing_federation_envoy`
+    - :ref:`establishing_federation_experiment_manager`
+
+Follow the procedure in the director-based workflow to become familiar with the setup required and APIs provided for each role in the federation: *Director manager*, *Collaborator manager*, and *Experiment manager (data scientist)*.
+
+.. note::
+    The Open Federated Learning (|productName|) interactive Python API enables the Experiment manager (data scientists) to define and start a federated learning experiment from a single entry point: a Jupyter\*\  notebook or a Python\*\  script.
+
+    See :doc:`director_based_workflow.interactive_api` for details.
+
+An overview of this workflow is shown below.
+
+.. figure:: ./source/openfl/static_diagram.svg
+
+.. centered:: Overview of the Director-Based Workflow
+
+
+.. # Copyright (C) 2020-2021 Intel Corporation
+.. # SPDX-License-Identifier: Apache-2.0
+
+
+.. _establishing_federation_director:
+
+Director Manager: Set Up the Director
+=====================================
+
+The *Director manager* sets up the *Director*, which is the central node of the federation.
+
+    - :ref:`optional_step_create_pki_using_step_ca`
+    - :ref:`step0_install_director_prerequisites`
+    - :ref:`step1_start_the_director`
+
+.. _optional_step_create_pki_using_step_ca:
+
+OPTIONAL STEP: Create PKI Certificates Using Step-CA (Optional)
+--------------------------------------------------------
+
+The use of mutual Transport Layer Security (mTLS) is recommended for deployments in untrusted environments to establish participant identity and to encrypt communication. You may either import certificates provided by your organization or generate certificates with the :ref:`semi-automatic PKI <semi_automatic_certification>` provided by |productName|.
+
+.. _step0_install_director_prerequisites:
+
+STEP 0: Install Open Federated Learning (|productName|)
+-------------------------------------------------------
+
+Install |productName| in a virtual Python\*\  environment. See :ref:`install_package` for details.
+
+.. _step1_start_the_director:
+
+STEP 1: Start the Director
+--------------------------
+
+Start the Director on a node with at least two open ports. See :ref:`openfl_ll_components` to learn more about the Director entity.
+
+1. Create a Director workspace with a default config file.
+
+    .. code-block:: console
+        fx director create-workspace -p path/to/director_workspace_dir
+
+ This workspace will contain received experiments and supplementary files (Director config file and certificates).
+
+2. Modify the Director config file according to your federation setup.
+
+ The default config file contains the Director node FQDN, an open port, path of certificates, and :code:`sample_shape` and :code:`target_shape` fields with string representation of the unified data interface in the federation.
+
+3. Start the Director.
+
+ If mTLS protection is not set up, run this command.
+
+    .. code-block:: console
```

Comment:
Please, add blank line

btw, for now it looks like:
![image](https://user-images.githubusercontent.com/89856652/145553587-ec1224b7-6690-4afb-8c3d-4ddde87957a9.png)